packages/sommelier.rb

```diff
@@ -238,6 +300,12 @@ def self.build
             # Return or exit depending upon whether script was sourced.
             (return 0 2>/dev/null) && return 0 || exit 0
           fi
+          if [[ "$(uname -m)" == i686 ]]; then
+            echo "Sommelier is not supported on i686."
+            # Return or exit depending upon whether script was sourced.
+            (return 0 2>/dev/null) && return 0 || exit 0
+          fi
```

Comment:
Should this be at the top of the script?

Reply:
I figured the container check is most important.